Sources/CardPayments/CardClient.swift

```diff
@@ -113,19 +113,15 @@ public class CardClient: NSObject {
                         return
                     }
                 
-                    analyticsService?.sendEvent("card-payments:3ds:confirm-payment-source:challenge-required")
+                    analyticsService?.sendEvent("card-payments:approve-order:auth-challenge-required")
```

Comment:
```suggestion
                    analyticsService?.sendEvent("card-payments:approve-order:payer-action-required")
```
I wonder if we should start aligning our naming conventions with PPaaS more? I've been back and forth for a while, but it may be helpful for migrating merchants with direct integrations to our SDK since they'll be familiar with the terminology.

It may be out of scope for this PR, but it applies throughout the SDK e.g. we could rename `CardClient.approveOrder()` to `CardClient.confirmPaymentSource()` etc. Does that sound worthwhile?

Reply:
It was a bit confusing. For some reason we had both in iOS, confirm-payment-source events and approve-order final events.
Somewhere down the line, there was confusion.

I thought about the naming and it'd be a bit confusing to have convention of API name because for instance, PayPal web module just presents the pages.

Reply:
Right yeah it's the same for us on Android. PayPal always shows an auth challenge. On the other hand, for Android Card we have a 2-step process since not all payment source confirmations require 3DS. They may succeed on the first `CardClient.approveOrder()`. It's also a 2-step process on Android mostly because of process kill.